docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    restart: always
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+    volumes:
+    - dbvolume1:/var/lib/postgresql/data/pgdata
+    ports:
+    - "5432"
+  redis:
+    image: redis
+  web:
+    build:
```

Comment:
You might include a (commented-out) line in here showing how someone could change the `command` to run `python manage.py runserver` instead, for testing in local dev.

Reply:
I have some tasks in the fabfile that use "docker-compose run" for this sort of thing.